src/js/components/Box/doc.js

```diff
@@ -54,6 +54,7 @@ the column axis when contained in a Grid.`
         color: PropTypes.string,
         dark: PropTypes.bool,
         image: PropTypes.string,
+        position: PropTypes.string,
```

Comment:
I'm wondering when we decide to just have callers do things via `style` vs. adding more properties. For example, this could be done via `style={{ backgroundPosition: 'center center' }}`, which is not too much heavier than `background={{ position: 'center center' }}`. Thoughts?

Reply:
We can definitely do that. but we would need to remove the `center center` that is added there today right? Also, I'm a little confused on why we have opacity available today. Isn't that the same situation as background position?

Reply:
We do opacity via T-shirt sizing, small, medium, large.

Our 'center center' won't conflict because any caller's inline style will come after whatever we set.